csc-overrides/partials/announcement.html

```diff
@@ -1,5 +1,5 @@
 {# Put the announcement, in HTML, right under this comment! #}
-<p>You are cordially invited to our weekly research support coffee hour on Zoom. <a href="https://ssl.eventilla.com/event/PP4WB" target="_blank">Click here for more information!</a></p>
+<p>Please note: Puhti will have a service break on Tuesday March 7 from 8:00 to 17:00.</p>
```

Comment:
```suggestion
<p>🚧 Please note: Puhti will have a service break on Tuesday March 7 from 8:00 to 17:00. 🚧</p>
```
Could a couple of "under construction" emojis boost the noticeability a bit?
![image](https://user-images.githubusercontent.com/50655931/221865823-35d801dc-17fc-4853-8da8-e6d67d53a2cf.png)

Reply:
I'll just reply to my own question: Why yes, yes it could! What a wonderful idea! Please commit this suggestion right away!